src/App/App.js

```diff
@@ -83,9 +111,14 @@ class App extends Component {
             <Route
               exact
               path="/"
-              render={() => <section className="wrapper">
-                <Movies movies={this.state.allMovies} getMovie={this.getMovie}/>
-              </section>}
+              render={() => {
+                return (
+                  <section className="wrapper">
+                    <SearchBar findMovie={this.findMovie}/>
+                    <Movies movies={this.state.allMovies} getMovie={this.getMovie}/>
+                  </section>
+                )
+              }}
```

Comment:
This is where I bring in the SearchBar component and pass the function to find the movie so that the child component has access to it.